utils/storybook/CanvasProviderDecorator.tsx

```diff
@@ -5,11 +5,21 @@ import {
   createCanvasTheme,
 } from '@workday/canvas-kit-labs-react-core';
 import {object} from '@storybook/addon-knobs';
+import {makeDecorator} from '@storybook/addons';
 
 const label = 'theme';
 
-export default (storyFn: () => React.ReactNode) => (
-  <CanvasProvider theme={createCanvasTheme(object(label, defaultCanvasTheme))}>
-    {storyFn()}
-  </CanvasProvider>
-);
+export default makeDecorator({
+  name: 'withCanvasProviderDecorator',
```

Comment:
Nit: What is the `with` here for?

Reply:
the documentation suggesting naming things like, will fix!